packages/plugins/mosquitto/mosquitto.spec

```diff
@@ -0,0 +1,471 @@
+%global with_tests 0
+%bcond_with websockets
+%bcond_with cjson
```

Comment:
I think we should have cjson support enabled (it is required for `mosquitto_ctrl` tool and the dynamic security plugin), but that needs cjson to be built, which I didn't do yet.